modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-cluster-wide-migration_{context}"]
+= About cluster-wide migration
+
+Migrating from {SMProductName} {SMv2Version} to {SMProduct} 3.0 can be performed using one of the following methods:
+
+* Applying the `istio.io/rev` label
+* Applying the `istio-injection=enabled` label
+* Performing a simple migration using the `istio-injection=enabled` label; however, the simple migration method should not be used in a production environment because there will be traffic disruptions.
+
+During the migration process, two cluster-wide control planes run in the same namespace. One control plane is associated with the 2.6 installation and the other one is associated with the 3.0 installation. You must carefully plan out the migration steps in order to avoid possible conflicts between the two control planes.
+
+== Control planes and root certificate
+
+During the migration, the control plane associated with the 2.6 installation and the control plane associated with the 3.0 installation must share the same root certificate. This can be achieved by installing the 3.0 control plane to the same namespace as 2.6 control plane. The migration procedures show how to verify the root certificate is shared.
+
+== Control planes, discovery selectors, and namespace access
+
+Both control planes must have access to all namespaces in the mesh. During the migration, some proxies will be controlled by the 3.0 control plane while other proxies will remain controlled by the 2.6 control plane. To ensure that mesh communication works during the migration, both control planes must be aware of the same set of services. 
+
+Discovery selectors specify the set of namespaces that comprise the mesh. If you are using discovery selectors, you must ensure that the `discoverySelectors` label defined in the {SMProduct} 3.0 {istio} resource match the namespaces that comprise the {SMProduct} 2.6 mesh. 
+
+If the 2.6 control plane uses discovery selectors that were automatically generated by the control plane, such as `maistra.io/member-of` label, the labels are removed during the migration process. To use discovery selectors in the 3.0 installation you must apply new labels to the application namespaces. See "Scoping the service mesh with DiscoverySelectors" for more information.
```

Comment:
```suggestion
In the {SMProduct} 3.0, you can control the service discovery via discovery selectors. If you are using discovery selectors, you must ensure that the `discoverySelectors` expression defined in the {SMProduct} 3.0 {istio} resource match the namespaces that comprise the {SMProduct} 2.6 mesh. You may need to add additional labels onto your OpenShift Service Mesh 2.6 application namespaces to ensure that they are captured by your OpenShift Service Mesh 3.0. See "Scoping the service mesh with DiscoverySelectors" for more information.

Note that automatically created label `maistra.io/member-of` by {SMProduct} 2.6 can't be used as it will be removed during the migration.
```

Reply:
done